examples/tox21/tox21_graph_conv.py

```diff
@@ -20,35 +21,13 @@
 metric = dc.metrics.Metric(
     dc.metrics.roc_auc_score, np.mean, mode="classification")
 
-# Number of features on conv-mols
-n_feat = 75
 # Batch size of models
 batch_size = 50
-graph_model = dc.nn.SequentialGraph(n_feat)
-graph_model.add(dc.nn.GraphConv(64, n_feat, activation='relu'))
-graph_model.add(dc.nn.BatchNormalization(epsilon=1e-5, mode=1))
-graph_model.add(dc.nn.GraphPool())
-graph_model.add(dc.nn.GraphConv(64, 64, activation='relu'))
-graph_model.add(dc.nn.BatchNormalization(epsilon=1e-5, mode=1))
-graph_model.add(dc.nn.GraphPool())
-# Gather Projection
-graph_model.add(dc.nn.Dense(128, 64, activation='relu'))
-graph_model.add(dc.nn.BatchNormalization(epsilon=1e-5, mode=1))
-graph_model.add(dc.nn.GraphGather(batch_size, activation="tanh"))
-
-model = dc.models.MultitaskGraphClassifier(
-    graph_model,
-    len(tox21_tasks),
-    n_feat,
-    batch_size=batch_size,
-    learning_rate=1e-3,
-    learning_rate_decay_time=1000,
-    optimizer_type="adam",
-    beta1=.9,
-    beta2=.999)
-
-# Fit trained model
-model.fit(train_dataset, nb_epoch=10)
+
```

Comment:
Couple requests here:

1. Let's move the `TensorGraph` graph-conv example to a new example file. We have a number of users of the existing graph-convs, so we should do a slow phase-out of the previous graph-convs over the next couple version bumps.
2. For the example, let's explicitly define the graph convs here instead of using the pre-built function.